### PR TITLE
Added a script to set the TLSN version number of the relevant crates

### PR DESCRIPTION
**Pull request by heeckhau**
Follow up of https://github.com/tlsnotary/tlsn/pull/642

### CODE REVIEW

**Comment by yuroitaki** [comment]
noice~ btw will this bump the version in notary's [openapi.yaml](https://github.com/tlsnotary/tlsn/blob/main/crates/notary/server/openapi.yaml#L6) too :P ?

**Comment by heeckhau** [comment]
It does not, it only changes the `Cargo.toml` files

**Review by yuroitaki — COMMENTED** [review]
Great stuff!

File: crates/notary/server/openapi.yaml [hunk 1]
```diff
@@ -1,169 +1,165 @@
...
-        "200":
+        '200':
           description: Ok response from server
           content:
             text/plain:
               schema:
                 type: string
-                example: "Ok"
-        "401":
+                example: Ok
```
[the remark this answers is not in the document]
Comment (heeckhau) [reply crates/notary/server/openapi.yaml:29]: This was autoformat

File: set_tlsn_version.rs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+---
+
+use clap::Parser;
+use serde_yaml::Value;
+use std::fs::{self, read_to_string};
+use std::path::Path;
+use toml_edit::{value, DocumentMut};
+use walkdir::WalkDir;
+
+#[derive(Parser)]
```
[the remark this answers is not in the document]
Comment (heeckhau) [reply set_tlsn_version.rs:26]: I've added a comment.
Note that `--help` also shows this

**Review by yuroitaki — APPROVED** [review]

**Pull request merged**